src/site/content/en/blog/yahoo-japan-news-bfcache/index.md

```diff
@@ -0,0 +1,161 @@
+
+## Impact of bfcache in numbers
+
+To measure the impact of bfcache, Yahoo! JAPAN News conducted an A/B test for 2 weeks, where they served a version of their pages with their bfcache fixes to one group, and a version without bfcache to another. They picked the URL paths with a significant amount of traffic so that the test could achieve meaningful results. There is no visual or functional difference between the 2 versions.
+
+Here's a video comparing the website with bfcache and without bfcache. You can see that the website with bfcache enabled loads significantly faster during a back or forward navigation.
+
+<figure>
+  {% Video src="video/jL3OLOhcWUQDnR4XjewLBx4e3PC3/2qqiTs9XuedWvtjfKuMu.mp4", autoplay="true", loop="true", width="800", height="800", muted="true" %}
+</figure>
+
+What’s even more amazing is that the group with bfcache enabled had seen a **significant increase in page views and ads revenue**, especially on mobile devices.
+
+Here are details about the impact observed by Yahoo! JAPAN News with their bfcache A/B test. (Further information can be found in [their case study article](https://techblog.yahoo.co.jp/entry/2023072430429932/)).
+
+<div class="table-wrapper scrollbar">
+  <table>
+    <thead>
+      <td>
+        <strong>Metrics</strong>
+      </td>
+      <td>
+        <strong>Uplift % (mobile)</strong>
+      </td>
+      <td>
+        <strong>Uplift % (desktop)</strong>
+      </td>
+    </thead>
+    <tbody>
+      <tr>
+        <td>bfcache hit rate</td>
+        <td>
+          <strong>+54.03 points</strong> (0.04% → 54.07%)
+        </td>
+        <td>+47.28 points (0.02% → 47.30%)</td>
+      </tr>
+      <tr>
+        <td>Page views</td>
+        <td>
+          <strong>+2.26%</strong>
+        </td>
+        <td>+0.65%</td>
+      </tr>
+      <tr>
+        <td>Ads revenue</td>
+        <td>
+          <strong>+9.0%</strong>
+        </td>
+        <td>+0.6%</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+When web pages become instant with bfcache, the users tend to stay on page longer, thus increasing the views on ads, leading to the increase of ads revenue.
+
+However, we think there is more to bfcache than just making the page instant.
+
+## bfcache enhances seamless user experience on the website
+
+bfcache is not just about making the pages instant. When the pages are instant, **the website navigation feels more seamless.**
+
+In Yahoo! JAPAN News, one of the major user journey is as follows:
+
+1. Go to the article list
+2. Click on one article to read
+3. Go back to the article list
+4. Click on another article to read
+
+Before bfcache, when users finished reading an article in (step 2), they had to wait for the article list page to load. This could be a friction factor for users who just want to go back to the article list to pick out another article to read.
```

Comment:
```suggestion
Before bfcache, when users finished reading an article in (step 2), they had to wait for the article list page to load again. This could be a friction factor for users who just want to go back to the article list to pick out another article to read.
```